networking/aws_load_balancer_operator/multiple-ingress-through-single-alb.adoc

```diff
@@ -6,6 +6,6 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can route the traffic to different services that are part of a single domain through a single AWS Load Balancer (ALB). Each Ingress resource provides different endpoints of the domain.
+You can route the traffic to different services that are part of a single domain through a single AWS Load Balancer. Each Ingress provides different endpoints of the domain.
```

Comment:
I almost feel like resource should be left in here or add something else like "class" or "instance" because it specifies Ingress. Just a thought.